streaming/src/reliability_helper.h

```diff
@@ -0,0 +1,68 @@
+#ifndef RAY_STREAMING_RELIABILITY_HELPER_H
+#define RAY_STREAMING_RELIABILITY_HELPER_H
+#include "channel/channel.h"
+#include "data_reader.h"
+#include "data_writer.h"
+#include "reliability/barrier_helper.h"
+#include "util/config.h"
+
+namespace ray {
+namespace streaming {
+
+class ReliabilityHelper;
+class DataWriter;
+class DataReader;
+
+class ReliabilityHelperFactory {
+ public:
+  static std::shared_ptr<ReliabilityHelper> GenReliabilityHelper(
```

Comment:
Factory method should be idiomatic, like `CreateReliabilityHelper`.

Reply:
abbr should be used sparingly. It's hard to know what `Gen` means

Reply:
done.